price-feeder/oracle/provider/websocket_controller.go

```diff
@@ -285,6 +285,9 @@ func (conn *WebsocketConnection) close() {
 
 	conn.logger.Debug().Msg("closing websocket")
```

Comment:
shall we have this log after the `conn.client == nil` check?

Reply:
When the client is nil it just means the socket has already been closed and there will be a corresponding error message for that event.